test/e2e/config/docker.yaml

```diff
@@ -237,6 +237,7 @@ intervals:
   default/wait-machine-pool-upgrade: ["5m", "10s"]
   default/wait-nodes-ready: ["10m", "10s"]
   default/wait-machine-remediation: ["5m", "10s"]
+  default/check-upgrade-no-rollout: ["10m", "30s"]
```

Comment:
I saw this was dropped and replaced by a const. Is there a particular reason for that? 
this makes the test less usable from providers...

Reply:
I'm not sure if we need a configurable timeout for providers to wait for longer until nothing happens.

But if we think it's worth a configuration option, fine for me to re-add it :)

Reply:
I am fine with re-adding it. Dropped it as it was pointed that it might no benefit from being configurable (https://github.com/kubernetes-sigs/cluster-api/pull/6184#discussion_r811642262):
>Not sure if we want to / have to make this configurable. This should be mainly a matter of how fast the machine controller would trigger a rollout and this should be the same for all providers